tools/update-timezone.mjs

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env node
+// Usage: tools/update-timezone.mjs
+import { execSync, spawnSync } from 'node:child_process';
+import { renameSync } from 'node:fs';
+import { exit } from 'node:process';
+
+const fileNames = [
+  'zoneinfo64.res',
+  'windowsZones.res',
+  'timezoneTypes.res',
+  'metaZones.res',
+];
+const dirs = spawnSync(
```

Comment:
Is there a reason to use child process instead of fs?

Reply:
Updated!